composer.json

```diff
@@ -12,7 +12,7 @@
   "require": {
     "php": ">=5.4.0",
     "brain/monkey": "^2.0.0",
-    "mockery/mockery": "^0.9.9"
+    "mockery/mockery": "~1.0"
```

Comment:
More as question for me? It is not more solid to use `^1` so that we gat always a version higher or equal that 1*.

Reply:
Now I have found the hwo to https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

* The ~ operator is best explained by example: ~1.2 is equivalent to >=1.2 <2.0.0
* The ^ operator behaves very similarly but it sticks closer to semantic versioning, and will always allow non-breaking updates.

Reply:
The tilde operator is a bit ambiguous as it behaves different on whether you define a concrete miner version or not. `~1.0` is equal to `>=1.0.0 <2.0.0` whereas `~1.0.0` is equal to `>=1.0.0 <1.1.0`. https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

Therefore the constraint is okay, but `^1.0` would make things more clear I guess.